app/src/main/java/io/gnosis/safe/ui/StartActivity.kt

```diff
@@ -58,6 +58,16 @@ class StartActivity : BaseActivity(), SafeOverviewNavigationHandler {
         setupNav()
 
         handleNotifications(intent)
+
+        if (settingsHandler.appStartCount <= 1) {
+            setupPasscode()
```

Comment:
what if user already has passcode in place? 
Also `appStartCount` is used as a counter to decide if rate the app dialog should be shown. So it will be set to 0 periodically

Reply:
Can I reuse `showPasscodeBanner()` instead of the appStartCount?

Reply:
Is there a safe way to check whether a passcode is set already? WIll there be an error from the encryptionManager, when I try to `setupPassword()` when it is locked? 

To me it looks like `setupPassword()` will fail, if there is already a password and it was not given to `setupPassword()`